lib/galaxy/tool_util/biotools/interface.py

```diff
@@ -21,8 +21,8 @@ class BiotoolsEntry:
     def from_json(from_json: Dict[str, Any]) -> 'BiotoolsEntry':
         entry = BiotoolsEntry()
         entry.biotoolsID = from_json["biotoolsID"]
-        entry.topic = from_json["topic"]
-        entry.function = from_json["function"]
+        entry.topic = from_json.get("topic", [{}])
+        entry.function = from_json.get("function", [{}])
```

Comment:
This is the correct fix:

```suggestion
        entry.topic = from_json.get("topic", [])
        entry.function = from_json.get("function", [])
```

Reply:
Thanks!

@bgruening Then I am committing this (instead of [what you suggested](https://github.com/usegalaxy-eu/galaxy/pull/179#discussion_r1272260868)) and merging.